src/module.c

```diff
@@ -12388,28 +12388,30 @@ int moduleLoad(const char *path, void **module_argv, int module_argc, int is_loa
 /* Unload the module registered with the specified name. On success
  * C_OK is returned, otherwise C_ERR is returned and errmsg is set
  * with an appropriate message. */
-int moduleUnload(sds name, const char **errmsg) {
+int moduleUnload(sds name, const char **errmsg, int forced_unload) {
```

Comment:
please add a comment for this new argument, like: only forcefully unload this module if it is certain that it has not yet been in use (e.g., immediate unload on failed load).